src/redux/Actions.ts

```diff
@@ -271,12 +271,12 @@ export const cookieCleanup: ActionCreator<
 
   // Show notifications after cleanup
   if (
-    setOfDeletedDomainCookies.size > 0 &&
+    setOfDeletedDomainCookies.length > 0 &&
     getSetting(getState(), 'showNotificationAfterCleanup')
   ) {
     const notifyMessage = browser.i18n.getMessage('notificationContent', [
       recentlyCleaned.toString(),
-      Array.from(setOfDeletedDomainCookies).join(', '),
+      setOfDeletedDomainCookies.join(', '),
```

Comment:
Why is this needed?

Reply:
This was because when I was reading the output the cadLog gave, it only returned `{}` when passed in as a Set.  As such I modified it in `returnSetOfOpenTabDomains` to return an array instead of a Set.

It doesn't necessarily need to be changed, but if an array is already being returned, there's no point of converting an array into an array again.

Reply:
Marking as resolved per above comment, unless changes in `returnSetOfOpenTabDomains` should be renamed now that it isn't returning a set...

Reply:
In 342cc64 I've renamed `returnSetOfOpenTabDomains` to `returnContainersOfOpenTabDomains`  
The function now returns an object, with the container id as key and an array of open tab domains.